e2e/capl-cluster-flavors/kubeadm-full-capl-cluster/assert-konnectivity-resources.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: konnectivity-agent
+  namespace: kube-system
+status:
+  availableReplicas: 3
+  readyReplicas: 3
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: konnectivity-server
+  namespace: kube-system
+status:
+  currentNumberScheduled: 1
+  desiredNumberScheduled: 1
+  numberAvailable: 1
+  numberMisscheduled: 0
+  numberReady: 1
```

Comment:
For konnectivity, I would also like us to have a test where we try to get logs for any pod and make sure we are able to fetch logs. So something like `kubectl get logs pod/csi-linode-controller-0 -n kube-system` and make sure the command runs successfully and doesn't hang or error out.

Reply:
So should I get logs for every pod or are there any particular ones?

Reply:
I've done this test as well. It's up and running.